src/Create.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+// we will redirecte the user after finish adding the blog to another page using hook
+import { useHistory } from 'react-router-dom';
+
+const Create = () => {
+    const [title,setTitle] = useState('');
+    const [body,setBody] = useState('');
+    const [author, setAuthor] = useState('mario')
+    // loading state when we add the blog
+    const [isPending, setIsPending] = useState(false);
+    const history = useHistory();
+
+    const handleSubmit = (e) => {
+        // Prevent page refresh on submit
+        e.preventDefault();
+        // creating a blog object that will carry the submitted data and send it to our db
+        const blog = {title, body, author};
+        setIsPending(true);
+        // creating post request to the json server
+        fetch('http://localhost:8000/blogs', {
+            method: 'POST',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/7bel6/b-blog/pullRequest?prid=7919486)
```suggestion
            method: "POST",
```